junit-jupiter-params/src/test/java/org/junit/jupiter/params/ParameterizedTestIntegrationTests.java

```diff
@@ -219,6 +220,17 @@ void executesLifecycleMethods() {
 		// @formatter:on
 	}
 
+	@Test
+	void truncatesArgumentsThatExceedMaxLength() {
+		var results = EngineTestKit.engine(new JupiterTestEngine()) //
+				.configurationParameter(ParameterizedTestExtension.ARGUMENT_MAX_LENGTH_KEY, "2") //
+				.selectors(selectMethod(TestCase.class, "testWithCsvSource", String.class.getName())) //
+				.execute();
+		results.testEvents().assertThatEvents() //
+				.haveExactly(1, event(displayName("[1] argument=f…"), started())) //
```

Comment:
So do we consider that `…` counts as a character? Personally, with a max length of _two_ I'd naturally expect `fo…`. But I may be the only one 🤷‍♀️

Reply:
TBH I was myself torn on this question. @junit-team/junit-lambda Thoughts?

Reply:
If I specify a max char count, I expect to get back a string of the given max length. For max = 5 and `...` it'd yield:

```
   "123" = "123"
  "1234" = "1234"
 "12345" = "12345"
"123456" = "12..."
```

Reply:
I'm using the Unicode ellipsis character `…` which is just one character so it would be `1234…` for `123456` in your example.

Reply:
Ah, I see. Nice trick to retain two characters from the source string. I'll update the example above accordingly.

Gist: a given max char count should not be exceeded. It's max. Not max+n. ;-)

Reply:
> Gist: a given max char count should not be exceeded. It's max. Not max+n. ;-)

Reply:
>Gist: a given max char count should not be exceeded. It's max. Not max+n.

I agree with that.

Reply:
So be it 😄 
But we should make that clear in the documentation then.